### PR TITLE
Revert "loader: revert #1544 to reduce risks"

### PR DESCRIPTION
**Pull request by lance6716**
Reverts #1657

### CODE REVIEW

**Comment by lance6716** [comment]
the conflict is merged in https://github.com/pingcap/dm/pull/1685 😂 I'm requesting https://github.com/ti-community-infra/tichi/issues/539

**Comment by lance6716** [comment]
@lichunzhu PTAL

File: loader/loader.go [hunk 1]
```diff
@@ -27,6 +26,8 @@ import (
 	"sync"
 	"time"
 
+	"golang.org/x/sync/errgroup"
```
Comment (lichunzhu) [inline loader/loader.go:29]: Combine this to L42~48

**Review by lichunzhu — COMMENTED** [review]
Rest LGTM

File: loader/loader.go (hunk 1 above)
Comment (lance6716) [reply loader/loader.go:29]: make fmt is not good to sort imports 😂

**Review by lichunzhu — COMMENTED** [review]
/lgtm

**Pull request merged**